src/component/YourList.tsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-console */
+import React from 'react';
+import { Card, Button } from 'antd';
+import { EditOutlined, DeleteOutlined } from '@ant-design/icons';
+
+const { Meta } = Card;
+
+const YourList: React.FC = () => {
+  // Sample books data
+  const books = [
+    {
+      id: 1,
+      bookName: 'Book Title 1',
+      writerName: 'Author 1',
+      description: 'Description of the book 1',
+      coverImage: 'https://i.postimg.cc/sgxNjwVn/book-4600757-1280.jpg',
+    },
+    {
+      id: 2,
+      bookName: 'Book Title 2',
+      writerName: 'Author 2',
+      description: 'Description of the book 2',
+      coverImage: 'https://i.postimg.cc/sgxNjwVn/book-4600757-1280.jpg',
+    },
+
+  ];
+
+  const handleUpdate = (id: number) => {
+    console.log('Update book with id:', id);
+    
+  };
+
+  const handleDelete = (id: number) => {
+    console.log('Delete book with id:', id);
+  
+  };
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexWrap: 'wrap',
+        justifyContent: 'space-around',
+        padding: '2rem',
+      }}
+    >
+      {books.map((book) => (
```

Comment:
`AntD` doesn't have `List` or `Table`?

Don't you think `{books.map((book) => (` is too much unoptimized?